diff -urN quick-lounge-applet-2.12.0.orig/configure.in quick-lounge-applet-2.12.0/configure.in
--- quick-lounge-applet-2.12.0.orig/configure.in	2005-09-29 13:07:02.785590864 +0200
+++ quick-lounge-applet-2.12.0/configure.in	2005-09-29 13:08:00.612799792 +0200
@@ -73,7 +73,7 @@
 
 dnl ******************************
 
-ALL_LINGUAS="az bg bs ca cs de el en_CA en_GB es fi fr gl gu hr hu it ja ko ms nl no pa pl pt pt_BR ru rw sk sr sr@Latn sv uk zh_CN"
+ALL_LINGUAS="az bg bs ca cs de el en_CA en_GB es fi fr gl gu hr hu it ja ko ms nb nl pa pl pt pt_BR ru rw sk sr sr@Latn sv uk zh_CN"
 GETTEXT_PACKAGE=quick-lounge-applet
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE", [GetText Package])
 AC_SUBST(GETTEXT_PACKAGE)
