diff -urN aa/quick-lounge-applet-2.1.1/configure.in quick-lounge-applet-2.1.1/configure.in
--- aa/quick-lounge-applet-2.1.1/configure.in	2004-03-30 19:31:54.000000000 +0200
+++ quick-lounge-applet-2.1.1/configure.in	2004-03-31 16:49:16.854217512 +0200
@@ -70,7 +70,7 @@
 
 dnl ******************************
 
-ALL_LINGUAS="az ca cs de el es fr hr it ja ko ms nl no pl pt pt_BR sk sr sr@Latn sv zh_CN"
+ALL_LINGUAS="az ca cs de el es fr hr it ja ko ms nb nl pl pt pt_BR sk sr sr@Latn sv zh_CN"
 GETTEXT_PACKAGE=quick-lounge-applet
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE", [GetText Package])
 AC_SUBST(GETTEXT_PACKAGE)
